DevCycle.SDK.Server.Local/Api/EventQueue.cs

```diff
@@ -62,127 +68,92 @@ public void RemoveFlushedEventsSubscriber(EventHandler<DVCEventArgs> flushedEven
         {
             FlushedEvents -= flushedEventsSubscriber;
         }
-        
+
+        private async Task<Tuple<FlushPayload, RestResponse>> GetPayloadResult(FlushPayload flushPayload)
+        {
+            return new Tuple<FlushPayload, RestResponse>(flushPayload, await dvcEventsApiClient.PublishEvents(flushPayload.Records));
+        }
+
         public virtual async Task FlushEvents()
         {
-            // cancel pending queued FlushEvents
-            tokenSource.Cancel();
-            
-            await EventQueueSemaphore.WaitAsync(0);
```

Comment:
We still need this based on the fact we can't use localbucketing in an async method.